docs/development/modules.md

```diff
@@ -115,17 +114,17 @@ It's recommended to use batch ingestion tasks to validate your implementation.
 The segment will be automatically rolled up to Historical note after ~20 seconds.
 In this way, you can validate both push (at realtime process) and pull (at Historical process) segments.
 
-* DataSegmentPusher
+#### DataSegmentPusher
 
 Wherever your data storage (cloud storage service, distributed file system, etc.) is, you should be able to see one new file: `index.zip` (`partitionNum_index.zip` for HDFS data storage) after your ingestion task ends.
 
-* DataSegmentPuller
+#### DataSegmentPuller
 
 After ~20 secs your ingestion task ends, you should be able to see your Historical process trying to load the new segment.
 
 The following example was retrieved from a Historical process configured to use Azure for deep storage:
 
-```
+```txt
 2015-04-14T02:42:33,450 INFO [ZkCoordinator-0] org.apache.druid.server.coordination.ZkCoordinator - New request[LOAD: dde_2015-01-02T00:00:00.000Z_2015-01-03T00:00:00
```

Comment:
Some of these example logs have changed. It's hard to keep these logs up-to-date with the code and maintain them. Assuming they don't add a lot of value, should we just remove them?

Instead, we can also direct the developer to use the quickstart, if needed, which uses the `LocalDataSegmentPuller` and doesn't require any special setup.